src/main/java/teammates/common/util/Const.java

```diff
@@ -1291,6 +1291,8 @@ private Const() {
         public static final String COURSE_INSTRUCTOR_EXISTS =
                 "An instructor with the same email address already exists in the course.";
         public static final String COURSE_INSTRUCTOR_EDITED = "The changes to the instructor %s has been updated.";
+        public static final String COURSE_INSTRUCTOR_NO_INSTRUCTOR_DISPLAYED = "At least one instructor must be "
+                                                                            + "displayed to students.";
```

Comment:
Avoid custom indentation in code. We've changed out from using custom indentation to using 8 spaces additional indentation due to various reasons, such as having to introduce extra changes when the length of a line changes (eg. renaming a variable/method). As an example, refer to the line for `COURSE_INSTRUCTOR_EXISTS` just above. This applies to multiple areas.

Also, if using this approach, for this case it may be better to put the entire string in a line on its own, as opposed to breaking it into two lines.